src/swipe-navigation.ts

```diff
@@ -447,6 +447,12 @@ class swipeManager {
   }
 
   static #handleTouchStart(event: TouchEvent) {
+    if (event.touches.length > 1) {
+      this.#xDown = null;
+      this.#yDown = null;
```

Comment:
Are these two necessaries?
I think so, and I think you put them to clear the state after the first touch occurred, but I want to be sure I understood correctly.

Reply:
Yes because the touches doesn't happen simultaneously. So one Touch instance for first touch is created and reacts to swipe. So I am clearing first detected Touch instance